build-tools/create-pkg/create-pkg.targets

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\installers\create-installers.targets" />
+  <PropertyGroup>
+    <PkgInstallDir>/</PkgInstallDir>
+    <PkgInstallDir Condition="$([MSBuild]::IsOsPlatform(Linux))">/usr</PkgInstallDir>
+    <PayloadDir>$(OutputPath)/pkgarchive</PayloadDir>
+    <PkgOutputPath>$(OutputPath)/packages</PkgOutputPath>
+  </PropertyGroup>
+  <Target Name="_CopyFilesToPayloadDir"
+      DependsOnTargets="GetXAVersion" >
+    <PropertyGroup>
+      <XAFrameworkDir>$(PayloadDir)/Library/Frameworks/Xamarin.Android.framework/Versions/$(XAVersion)</XAFrameworkDir>
+      <MonoFrameworkExternalDir>$(PayloadDir)/Library/Frameworks/Mono.framework/External</MonoFrameworkExternalDir>
+      <XBuildDir>$(XAFrameworkDir)/lib/xamarin.android/xbuild/Xamarin/Android</XBuildDir>
+      <XBuildFrameworkDir>$(XAFrameworkDir)/lib/xamarin.android/xbuild-frameworks/MonoAndroid</XBuildFrameworkDir>
+    </PropertyGroup>
+    <RemoveDir Directories="$(PayloadDir)" />
+    <Copy
+        SourceFiles="@(FrameworkFiles)"
+        DestinationFolder="$(XBuildFrameworkDir)/%(RecursiveDir)"
+    />
+    <Copy
+        SourceFiles="@(FirstFrameworkFiles)"
+        DestinationFolder="$(XBuildFrameworkDir)/$(FirstFrameworkVersion)/%(RecursiveDir)"
+    />
+    <Copy
+        SourceFiles="@(BCLFrameworkFiles)"
+        DestinationFolder="$(XBuildFrameworkDir)/$(BclFrameworkVersion)/%(RecursiveDir)"
+    />
+    <Copy
+        SourceFiles="@(MSBuildFiles);@(MSBuildFilesUnix)"
+        DestinationFolder="$(XBuildDir)/%(RecursiveDir)"
+    />
+    <Copy
+        SourceFiles="@(MSBuildFilesDarwin)"
+        DestinationFolder="$(XBuildDir)/Darwin"
+    />
+    <Copy
+        SourceFiles="@(MSBuildLibFiles)"
+        DestinationFolder="$(XBuildDir)/lib/%(RecursiveDir)"
+    />
+    <Copy
+        SourceFiles="@(MSBuildLibFilesDarwinHost)"
+        DestinationFolder="$(XBuildDir)/lib/host-Darwin"
+    />
+    <Copy
+        SourceFiles="@(VersionFiles)"
+        DestinationFolder="$(XAFrameworkDir)"
+    />
+    <Copy
+        SourceFiles="@(TargetsFiles)"
+        DestinationFolder="$(XAFrameworkDir)/lib/xamarin.android/xbuild/%(RecursiveDir)"
+    />
+  </Target>
+  <Target Name="_CreateSymbolicLinks"
+      DependsOnTargets="_CopyFilesToPayloadDir">
+    <Exec WorkingDirectory="$(PayloadDir)/Library/Frameworks/Xamarin.Android.framework/Versions"
+      Command="ln -fs $(XAVersion) Current"
+    />
+    <MakeDir Directories="$(MonoFrameworkExternalDir)/xbuild"/>
+    <Exec WorkingDirectory="$(MonoFrameworkExternalDir)/xbuild"
+      Command="ln -fs $([System.IO.Path]::GetFullPath('$(XAFrameworkDir)/lib/xamarin.android/xbuild/Novell')) ."
```

Comment:
This is almost certainly *not* what we want: `Path.GetFullPath($XAFrameworkDir)` will return `/some/random/path/to/xamarin.android/bin/BuildDebug/pkgarchive/...`, which won't be helpful. :-)

This should instead be a relative symlink.

We should also check to see if this symlink is required at all.

Reply:
This symlink is required for projects which import a `\Novell.MonoDroid.CSharp.targets` file (e.g. https://github.com/xamarin/monodroid-samples/blob/master/Button/DroidButton.csproj#L54) so I think it's best to keep this for backwards compatibility.

I'll take a look at fixing these symlinks to use relative paths, I didn't catch this while testing locally because the arbitrary fully qualified path did contain the required content.